File: src/main/java/jugglinglab/util/JuggleException.java
//
// JuggleException.java
//
// This is our own exception class, which we use to handle juggling-related
// problems that occur.
//
// Copyright 2002-2025 Jack Boyce and the Juggling Lab contributors
//

package jugglinglab.util;

public class JuggleException extends Exception {
  public JuggleException() {
    super();
  }

  public JuggleException(String s) {
    super(s);
  }
}

File: src/main/kotlin/jugglinglab/util/JuggleException.kt
//
// JuggleException.kt
//
// This is our own exception class, which we use to handle juggling-related
// problems that occur.
//
// Copyright 2002-2025 Jack Boyce and the Juggling Lab contributors
//

package jugglinglab.util

open class JuggleException : Exception {
    constructor() : super()

    constructor(s: String?) : super(s)
}
